extensions/dirvish-emerge.el

```diff
@@ -501,5 +501,20 @@ If in the first group move to the beginning of buffer."
                (backward-char 1))
              (goto-char (overlay-start (dirvish-emerge--get-group-overlay))))
     (goto-char (point-min))))
+
+;; Ideally this would be a toggle. But it is not possible to be
+;; inside a hidden group.
+(defun dirvish-emerge-hide-current-group ()
+  "Hide the current group."
+  (interactive)
+  (when-let ((group-overlay (dirvish-emerge--get-group-overlay))
+             (group-id (overlay-get group-overlay 'dirvish-emerge-group))
+             (group (nth (1- group-id) dirvish-emerge-groups)))
+    (overlay-put group-overlay 'before-string
+                 (dirvish-emerge--format-group-title
+                  (concat (nth 0 group) " (Hidden)")))
+    (overlay-put group-overlay 'invisible t)
+    (setf (nth 2 group) t)))
+
```

Comment:
Is it possible to toggle a group if the title/heading is separated from the group files? Something like I commented [here](https://github.com/alexluigit/dirvish/pull/68/commits/9da8520b1cf192f15b1fad4f752cc1634fabe036#r912615780)

Reply:
Yes, that is implemented now.

Reply:
Great! Could you replace that usage of `length<` with `(< (length group) 3)`? `length<` is only available on Emacs 28+.

Reply:
Done!